### PR TITLE
Fix test_auth_credentials.py

### PR DESCRIPTION
**Pull request by pehala**
- Run it also on Kuadrant
- Fix wrong asserts

### CODE REVIEW

File: testsuite/tests/kuadrant/authorino/identity/rhsso/test_auth_credentials.py [hunk 1]
```diff
@@ -12,33 +11,45 @@ def credentials(request):
...
+    """Add RHSSO identity to Authorization"""
+    authorization.identity.clear_all()
     authorization.identity.add_oidc("rhsso", rhsso.well_known["issuer"], credentials=Credentials(credentials, "Token"))
     return authorization
 
 
 def test_custom_selector(client, auth, credentials):
     """Test if auth credentials are stored in right place"""
     response = client.get("/get", headers={"authorization": "Token " + auth.token.access_token})
-    assert response.status_code == 200 if credentials == "authorization_header" else 401
```
Comment (averevki) [inline testsuite/tests/kuadrant/authorino/identity/rhsso/test_auth_credentials.py:25]: Wouldn't it be sufficient to just put brackets like this? 
`assert response.status_code == (200 if credentials == "authorization_header" else 401)`

File: testsuite/tests/kuadrant/authorino/identity/rhsso/test_auth_credentials.py [hunk 2]
```diff
@@ -12,33 +11,45 @@ def credentials(request):
 
 
 @pytest.fixture(scope="module")
-def authorization(rhsso, openshift, blame, route, module_label, credentials):
-    """Add RHSSO identity to AuthConfig"""
-    authorization = AuthConfig.create_instance(openshift, blame("ac"), route, labels={"testRun": module_label})
+def authorization(authorization, rhsso, credentials):
+    """Add RHSSO identity to Authorization"""
+    authorization.identity.clear_all()
```
Comment (jsmolar) [inline testsuite/tests/kuadrant/authorino/identity/rhsso/test_auth_credentials.py:16]: You don't need to clear identities here. `authorization` from rhsso conftest should be enough

Comment (pehala) [reply testsuite/tests/kuadrant/authorino/identity/rhsso/test_auth_credentials.py:16]: I do, due to parametrization

File: testsuite/tests/kuadrant/authorino/identity/rhsso/test_auth_credentials.py (hunk 1 above)
Comment (pehala) [reply testsuite/tests/kuadrant/authorino/identity/rhsso/test_auth_credentials.py:25]: It would, I think it is more readable with separate if

**Review by averevki — APPROVED** [review]

**Review by jsmolar — APPROVED** [review]

**Pull request merged**